backend/src/bundles/users/user.service.ts

```diff
@@ -49,10 +53,48 @@ class UserService implements Service {
     }
 
     public async update(
+        query: Record<string, number>,
+        payload: UserUpdateProfileRequestDto,
+    ): Promise<UserAuthResponseDto | null> {
+        try {
+            const userId = query['id'] as number;
```

Comment:
and you are anyway relying on presence id in this query, if there will be no id present this will drop exception

Reply:
not resolved